javascripts/downloads.js

```diff
@@ -0,0 +1,149 @@
+app.downloads = {};
+
+// app.downloads.url = 'https://developers.stage.redhat.com/download-manager/rest/available/';
+app.downloads.url = 'https://www.stage.jboss.org/download-manager/rest/available/';
```

Comment:
Is this the correct production URL? If it isn't we probably need it to be a config variable.